music21/scale/__init__.py

```diff
@@ -2878,6 +2940,36 @@ def __init__(self, tonic=None):
         # note: this changes the previously assigned AbstractDiatonicScale
         # from the DiatonicScale base class
         self._abstract = AbstractMelodicMinorScale()
+        
+        
+class JazzMinorScale(DiatonicScale):
+    '''
+    The jazz minor scale.
+    
+    A synthetic scale identical to the ascending melodic minor. Useful in
+    jazz, especially over harmonies that employ minor-major seventh chords.
+    >>> sc = scale.JazzMinorScale('e4')
```

Comment:
Great explanation, thanks. But a potential gotcha is that you need a blank line between the paragraph and the doctests (beginning with the Python prompt) for Sphinx to render this correctly.